docs/kbs.yaml

```diff
@@ -71,19 +71,37 @@ paths:
               schema:
                 $ref: '#/components/schemas/ErrorInformation'
 
-  /resources/key/<key_id>:
+  /resources/{repository}/{type}/{tag}:
```

Comment:
Maybe we should rename `resources` to `resource`?

Reply:
OK, I think so.